### ISSUE TITLE
butane explosions seem too deadly

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#66760]
### Describe the bug

I just had a character die from a butane can exploding in fire at their feet. This character was in RM13 armor and had the active defense bionic and had 209 HP but still died instantly. This seemed weird to me so I tested it in debug by setting my strength to 9999 and found that the butane explosion (0.5 liter canister of butane, 400 units of butane) did 30,000 damage to each limb. Oddly even an empty butane can still did over 1000 damage to each limb.

### Attach save file

n/a

### Steps to reproduce

1. set hp to an arbitrarily high number.
2. light something on fire and step into the fire (wear Rm13 or something to be immune to fire)
3. drop a butane can at your feet and observe the damage it deals.

### Expected behavior

A 0.5 liter canister of butane probably shouldn't be stronger than a mininuke.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.1848 (22H2)
- Game Version: b69ae48 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Bionic Slots [cbm_slots]
]

**Comment by RenechCDDA** [comment]
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/84619419/5e7adf55-5336-4bb1-bc11-bc3b6dd6f39a)

https://github.com/CleverRaven/Cataclysm-DDA/blob/96c7690569678241cc5c7177f1c27ac5413c8c02/data/json/items/tool/misc.json#L68

https://github.com/CleverRaven/Cataclysm-DDA/blob/ffe5a450999bd7a575165e6416848886e6d5c166/doc/JSON_INFO.md#L3111

It looks like casing_mass is far, far too high. The docs don't specify what unit it's supposed to be either. I would assume grams, as it's an **integer** value, but the calculations seem to suggest it is **kilograms** (you know, 1000 more). Reducing casing_mass to 1 still produces an explosion of 300 damage... like due to the fact `power` is... 4503.

or as the docs put it:

"Measure of explosion power in **grams of TNT equivalent explosive**, affects damage and range."

In other words, the tiny 600g butane can with ~240g of butane currently counts as 4.5kg of TNT.

Absurd.

**Comment by RenechCDDA** [comment]
Yes ok based on the history of #24171 casing_mass should definitely be grams.

Unfortunately setting the butane can's casing_mass to a reasonable value (250g) causes it to still deal tens of thousands of damage, even after reducing `power` to 120g TNT-equivalent.
